Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.cs

```diff
@@ -20,6 +24,22 @@ public partial class FirstPersonPlayer : FirstPersonCharacter
         public void Initialize()
         {
             SetupInput();
+            
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+            UnityEngine.SceneManagement.SceneManager.sceneUnloaded += arg0 => 
+            {
+                movementInputAction?.Disable();            
```

Comment:
Can you put all these `Disable` into the `OnOnDestroy` method instead?

Reply:
Actually Im planning to remove this altogether. this was my original idea to solve the Error Message I posted in the description, but was only working some of the times. Also I think `OnDisable` is called when `OnDestroy` is called. Will really have to look into this soon